packages/bierzo-wallet/src/logic/connection.unit.spec.ts

```diff
@@ -17,6 +17,7 @@ withChainsDescribe('Logic :: connection', () => {
     await getConnectionFor(firstChain.chainSpec);
 
     expect(ethereumConnectionSpy).toHaveBeenCalledTimes(1);
+    await disconnect();
```

Comment:
Why not move this in `afterEach` (disconnect once oer file) or `afterAll` (disconnect once per spec) like in other files? Makes it harder to forget when new tests come in